docs/modules/ROOT/pages/vshn-managed/postgresql/backup.adoc

```diff
@@ -0,0 +1,78 @@
+= How to manage backups
+
+Our xref:apiserver/how-to.adoc[AppCat API Server] allows managing backups for VSHN PostgreSQL instances. You can `watch`, `list` or `get` a specific backup with all its details. Other methods such as `delete` will be added in a subsequent API Server version.
```

Comment:
For an end-user documentation, that's too technical. These API verbs are a low-level implementation detail, not relevant to an end-user. Please rephrase this sentence to not include such low-level details.